### PR TITLE
Feat: Finding Largest element in an array function created

### PR DESCRIPTION
**Pull request by KrithikaSuvarna01**
Guys can you please review my code

### CODE REVIEW

File: array.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+function array_largest(arr){
```
Comment (hrithik-b01) [inline array.js:7]: Using array.sort function will cause some bugs. Please fix your code.

**Review by hrithik-b01 — CHANGES_REQUESTED** [review]

File: array.js (hunk 1 above)
Comment (KrithikaSuvarna01) [reply array.js:7]: Fixed!

Comment (hrithik-b01) [reply array.js:7]: Code looks good. Can you please add some documentation?

**Review by hrithik-b01 — APPROVED** [review]
LGTM.

**Review by pr-th-m — APPROVED** [review]
LGTM

**Pull request merged**